test/integration/load/load_test.go

```diff
@@ -131,21 +131,26 @@ func TestValidateState(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
 	defer cancel()
 
-	t.Log("Validating the state file")
-	validatorClient := validate.GetValidatorClient(*osType)
-	validator := validatorClient.CreateClient(ctx, clientset, config, namespace, *cniType, *restartCase)
+	if *osType == "windows" {
+		t.Log("Validating the windows state file")
+		windowsClient := &validate.WindowsClient{}
+		windowsClient = windowsClient.CreateClient(ctx, clientset, config, namespace, *cniType, *restartCase)
```

Comment:
no need to create an empty struct to then populate later, just use a constructing function. Change this
```
func (*WindowsClient) CreateClient(context.Context, ...)
```
to construct the WindowsClient itself:
```
func CreateWindowsClient(context.Context, ...) (*WindowsClient) {
    return &WindowsClient{
        clientset: clientset,
        config: config,
        ...
    }
}
```

Reply:
Cool, updated it with the rebase from your PR.